src/sandbox/sandbox_state.js

```diff
@@ -0,0 +1,45 @@
+import { PricingStrategy } from '../utility/PricingStrategy';
```

Comment:
double quotes for imports.. you could try `npm run lint:fix' for the common issues.

Reply:
Same as [comment](https://github.com/singnet/refactored-dapp/pull/110#discussion_r309244829)